### PR TITLE
🔧 MediaItem placeholder prop

### LINKED ISSUE #5505: Make ImageMedia more accessible for non-KodaDot projects
why do you need to access `darkMode` in `libs/ui`?
same for `placeholder` I would pass it as props instead.

Use case: 
I'm using `@kodadot1/brick` in a different project and I want to use custom placeholder

_Originally posted by @roiLeo in https://github.com/kodadot/nft-gallery/pull/5429#pullrequestreview-1369294260_

### PR DESCRIPTION
**Pull request by roiLeo**
## PR Type

- [x] Bugfix
- [x] Refactoring

## Context

- [x] Closes #5505 

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [ ] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

## Screenshot 📸

⏳ 

### <samp>🤖 Generated by Copilot at 9846b66</samp>

> _Oh we're the coders of the NFTs_
> _And we work hard to make them please_
> _We added a `placeholder` prop_
> _To show the right image on the top_

### CODE REVIEW

**Review by daiagi — APPROVED** [review]

File: components/gallery/GalleryItem.vue [hunk 1]
```diff
@@ -150,6 +152,9 @@ const canPreview = computed(() =>
     resolveMedia(nftMimeType.value)
   )
 )
+const placeholder = computed(() =>
+  isDarkMode ? '/placeholder.webp' : '/placeholder-white.webp'
```
Comment (Jarsen136) [inline components/gallery/GalleryItem.vue:156]:
```suggestion
  isDarkMode.value ? '/placeholder.webp' : '/placeholder-white.webp'
```

File: components/gallery/GalleryItemPreviewer.vue [hunk 2]
```diff
@@ -28,12 +29,16 @@ import { useVModel } from '@vueuse/core'
...
 const { nft, nftImage, nftAnimation, nftMimeType } = useGalleryItem()
+const { isDarkMode } = useTheme()
 
 const props = defineProps<{
   value: boolean
 }>()
 const emit = defineEmits(['input'])
 const isFullscreen = useVModel(props, 'value', emit)
+const placeholder = computed(() =>
+  isDarkMode ? '/placeholder.webp' : '/placeholder-white.webp'
```
Comment (Jarsen136) [inline components/gallery/GalleryItemPreviewer.vue:40]:
```suggestion
isDarkMode.value ? '/placeholder.webp' : '/placeholder-white.webp'
```

File: components/shared/gallery/NftCard.vue [hunk 3]
```diff
@@ -12,12 +13,17 @@ import type { NFT } from '@/components/rmrk/service/scheme'
...
 const { urlPrefix } = usePrefix()
+const { isDarkMode } = useTheme()
 
 const props = defineProps<{
   nft: NFT | CarouselNFT
   variant?: NftCardVariant
 }>()
 
+const placeholder = computed(() =>
+  isDarkMode ? '/placeholder.webp' : '/placeholder-white.webp'
```
Comment (Jarsen136) [inline components/shared/gallery/NftCard.vue:24]:
```suggestion
 isDarkMode.value ? '/placeholder.webp' : '/placeholder-white.webp'
```

**Review by Jarsen136 — COMMENTED** [review]
I'm think that if we could put the placeholder image into `useTheme` composable function. In this way, we could avoid some dumplicate logic code.

**Comment by roiLeo** [comment]
> I'm think that if we could put the placeholder image into `useTheme` composable function. In this way, we could avoid some duplicate logic code.

Thanks! 🤸‍♂️

**Comment by roiLeo** [comment]
🚀

**Pull request merged**